zoom_utils/constants.py

```diff
@@ -0,0 +1,10 @@
+DATE_TIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+DATE_FORMAT = '%Y-%m-%d'
```

Comment:
nit: make it configurable

Reply:
I don't think so it needs to configurable, because we are using this format only for internal calculation rather than display on the excel sheet.